components/library/enet/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		enet
+COMPONENT_VERSION=	1.3.14
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_FMRI=		library/security/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=	System/Libraries
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_PROJECT_URL=	http://enet.bespin.org
+# Development is on GitHub, but releases are not :(
+#COMPONENT_PROJECT_URL=	https://github.com/lsalzman/enet
+#COMPONENT_ARCHIVE_URL=	https://github.com/lsalzman/$(COMPONENT_NAME)/releases/download/$(COMPONENT_NAME)-$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
```

Comment:
I don't think this fact is worth documenting.

Reply:
Well... in case this component does get picked up, and in case future releases would go through Github, then maintainers of this recipe would have an easier life looking for current sources :)

Reply:
It was trivial for me to find this information on their site. We have several such components with development site/download site dichotomy and we don't document this. Drop it, please.

Reply:
ack, done